packages/cloudflare/src/utils/deduplicatePatterns.ts

```diff
@@ -10,16 +10,17 @@ export function deduplicatePatterns(patterns: string[]) {
 
 	// A value in the set may only occur once; it is unique in the set's collection.
 	// ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set
+	const uniquePatterns = [...new Set(patterns)];
+	for (const pattern of uniquePatterns) {
+		if (pattern.endsWith('*')) {
+			openPatterns.push(new RegExp(`^${pattern.replace(/(\*\/)*\*$/g, '[^*\n]*$')}`));
+		}
+	}
+
 	return [...new Set(patterns)]
```

Comment:
Could we also reuse `uniquePatterns` here?
```suggestion
	return uniquePatterns
```